src/tests/integration_tests/test_serialization.py

```diff
@@ -1,13 +1,16 @@
+import pytest
 from langchain.chat_models import ChatOpenAI
 
 from sherpa_ai.agents import Physicist
 from sherpa_ai.events import Event
 from sherpa_ai.orchestrator import EventType, Orchestrator, OrchestratorConfig
+from tests.fixtures.llms import get_llm
 
 
-def test_orchestrator():
+@pytest.mark.external_api
+def test_orchestrator_successful(get_llm):
     orchestrator = Orchestrator(OrchestratorConfig())
-    llm = ChatOpenAI(model_name="gpt-4")
+    llm = get_llm(__file__, test_orchestrator_successful.__name__)
```

Comment:
Looks like we were testing before with gpt-4 but no longer doing that. Is that intentional ?

Reply:
Yes. For testing purposes, we don't need GPT-4 (The code should function normally without GPT-4, though the performance may be worse.)